File: MyApplication3/app/src/main/java/com/example/myapplication3/MainActivity.java
package com.example.myapplication3;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private Button button;
    private TextView textView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button= (Button)findViewById(R.id.button);
        textView = (TextView)findViewById((R.id.textView6));

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(MainActivity.this,"HEllo Star",Toast.LENGTH_SHORT).show();
            }
        });


    }
}

File: kotlinApplilcation_basic/app/src/main/java/com/example/kotlinapplilcation_basic/MainActivity.kt
package com.example.kotlinapplilcation_basic

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val text:TextView =findViewById(R.id.textview);
        val button:Button=findViewById(R.id.button);
        button.setOnClickListener {
            Toast.makeText(this,"Hello Kotlin",Toast.LENGTH_SHORT).show();
            text.setText("Hello kotlin")
        };
    }
}